### DISCUSSION TITLE
Using cloudinary with supabase storage

### DISCUSSION THREAD

**Discussion by fergusmeiklejohn** [supabase/supabase#1902] [Questions] [answered]
Cloudinary Media Optimiser can point to an S3 or Google Cloud bucket. I think supabase storage is S3? Can we use cloudinary with supabase storage by pointing cloudinary at our (Supabase) s3 bucket?

**Comment by inian** [comment, marked as answer]
Hi @fergusmeiklejohn, can create a [public bucket](https://supabase.io/docs/guides/storage#accessing-objects) in Supabase Storage and point Cloudinary to it using the [fetch mode](https://cloudinary.com/documentation/fetch_remote_images)

**Reply by fergusmeiklejohn** [reply]
Thanks @inian 
I was thinking about Media Optimiser: https://cloudinary.com/documentation/media_optimizer
That requires you to point cloudinary to your S3 bucket, adding your access key and secret key. So I'm wondering if Supabase is spinning up an S3 bucket which we can point to?

**Reply by inian** [reply]
Try the proxy mode - https://cloudinary.com/documentation/media_optimizer_configuration#http_proxy That should work.
